_layouts/post.html

```diff
@@ -13,7 +13,7 @@ <h1 class="post-title">{{ page.title }}</h1>
     </section>
     <footer class="post-footer">
       <section class="share">
-        <a class="icon" href="http://twitter.com/share?text={{page.title | cgi_escape}}&amp;url={{site.url}}{{page.id}}" target="_blank">
+        <a class="icon" href="https://twitter.com/intent/tweet?url={{ site.url }}{{ page.id }}&text={{ page.title }}&via={{ site.twitter_username }}" target="_blank">
```

Comment:
Can you add the `| cgi_escape` filter to `page.title` here to make sure page title doesn't contain any invalid characters when the url is generated?